frontend/packages/erd-core/src/schemas/cliVersion/schemas.ts

```diff
@@ -1,9 +1,21 @@
 import * as v from 'valibot'
 
-export const cliVersionSchema = v.object({
+const webVersionSchema = v.object({
+  gitHash: v.string(),
+  date: v.string(),
+  displayedOn: v.literal('web'),
+})
+
+const innerCliVersionSchema = v.object({
   version: v.string(),
   gitHash: v.string(),
   envName: v.string(),
   isReleasedGitHash: v.boolean(),
   date: v.string(),
+  displayedOn: v.literal('cli'),
 })
+
+export const cliVersionSchema = v.union([
+  webVersionSchema,
+  innerCliVersionSchema,
+])
```

Comment:
used discriminated union👍
e897cc63d58e57b7eb5d2fcddae8f3667250f43a

Reply:
@MH4GF 
I've refactored cliVersion type definition, pls re-review👍

Reply:
Oh, I forgot I should refactor 😅
How about next PR?

<img width="400" alt="image" src="https://github.com/user-attachments/assets/4750d687-a64d-40f9-a037-b0436d5fdbb3" />

Reply:
@sasamuku Oh! OK, next PR will be fine!